CHANGELOG.md

```diff
@@ -0,0 +1,33 @@
+# Changelog
```

Comment:
I liked [cuSpatials change log](https://github.com/rapidsai/cuspatial/blob/branch-23.08/CHANGELOG.md#-breaking-changes-1) with the sections (with emotes :100: :sunglasses:  ):

With the points:
- Breaking Changes
- Bug Fixes
- Documentation
- New Features
- Improvements

I like keeping Dependencies (not sure how frequent we have Security updates)

https://docs.rapids.ai/resources/changelog/

Thoughts?